pkg/pidfile/pidfile.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strconv"
 )
 
+// PidFile file used as a pid
```

Comment:
I think to keep with the convention of keeping acronyms in uppercase, this should be PIDFile

The comment could read "A PIDFile is a file used to store the process ID of a running process"